Description: Upstream changes introduced in version 2.1.1-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 tellico (2.1.1-2) unstable; urgency=low
 .
   * Disable webcam support if libv4l is not present, fixing FTBFS on
     kfreebsd (Closes: #559731).
   * Add missing Build-depend on libksane-dev.
 .
 The person named in the Author field signed this changelog entry.
Author: Regis Boudin <regis@debian.org>
Bug-Debian: http://bugs.debian.org/559731

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- tellico-2.1.1.orig/CMakeLists.txt
+++ tellico-2.1.1/CMakeLists.txt
@@ -88,6 +88,8 @@ if(ENABLE_WEBCAM)
     macro_log_feature(LIBV4L_FOUND "libv4l" "Support for barcode scanning with a webcam" "http://hansdegoede.livejournal.com/3636.html" FALSE "" "Required for using a webcam")
     if(LIBV4L_FOUND)
         include_directories(${LIBV4L_INCLUDE_DIR})
+    else(LIBV4L_FOUND)
+        set(ENABLE_WEBCAM FALSE)
     endif(LIBV4L_FOUND)
 endif(ENABLE_WEBCAM)
 
